### PR TITLE
Responsive desktop feature 

### PR DESCRIPTION
**Pull request by 0zedd**
This feature includes media queries to make page responsive on larger screens

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -601,3 +794,477 @@ button:hover {
   left: 280px;
   top: 5110px;
 }
+
+@media screen and (min-width: 768px) {
+  .container {
+    width: 1440px;
+    display: flex;
+    flex-direction: column;
+    border: solid;
```
Comment (zieeco) [inline style.css]: **[REQUIRED CHANGES]**

- Please, kindly remove this border to match the Figma design

File: style.css [hunk 2]
```diff
@@ -601,3 +794,477 @@ button:hover {
   left: 280px;
   top: 5110px;
 }
+
+@media screen and (min-width: 768px) {
+  .container {
+    width: 1440px;
```
Comment (zieeco) [inline style.css]: **[REQUIRED CHANGES]**

- Please, kindly adjust your styling to ensure your width is not fixed but in percentage to remove the horizontal scroll behavior to make your portfolio responsive to improve the user experience below:


https://user-images.githubusercontent.com/59397159/172931993-d6a06864-c030-4e29-a19d-45c23ed0831d.mp4

File: style.css [hunk 3]
```diff
@@ -601,3 +794,477 @@ button:hover {
...
+  }
+
+  .contact_section {
+    margin-left: 260px;
+  }
+
+  .sec_icn {
+    margin-left: 210px;
+  }
+
```
Comment (zieeco) [inline style.css]: **[REQUIRED CHANGES]**

- Please, kindly ensure that **ALL** your social media icons have the same width to match the Figma design below:

*figma*
iconss

*your design*
icons

File: style.css [hunk 4]
```diff
@@ -601,3 +794,477 @@ button:hover {
...
+  .p {
+    display: flex;
+    font-size: 16px;
+    height: 50px;
+    width: 496px;
+    margin-left: 756px;
+    margin-right: 188px;
+    margin-top: 170px;
+  }
```
Comment (zieeco) [inline style.css:1050]: **[REQUIRED CHANGES]**

- Please, kindly ensure you've used the correct font family on all the paragraphs text to match the Figma design below:

*figma*
right-fonts

*your's*
right-font

File: style.css [hunk 5]
```diff
@@ -601,3 +794,477 @@ button:hover {
...
+    width: 203px;
+    margin-left: 756px;
+  }
+
+  button {
+    width: 149px;
+    margin-left: 756px;
+    margin-top: 40px;
+  }
+
```
Comment (zieeco) [inline style.css:1063]: **[REQUIRED CHANGES]**

- Please, kindly use the correct padding, and add the correct color and background to all the buttons to indicate the interaction of hover, active, enabled, and disabled to match the Figma design as you've used another background color and the other states of the buttons haven't been implemented yet . see the image below:

buton

File: style.css [hunk 6]
```diff
@@ -377,6 +518,27 @@ button {
...
+  padding: 12px;
+  width: 149px;
+  margin-left: 40px;
+  border-radius: 8px;
+  letter-spacing: 0.03em;
+}
+
 .about_section_container {
   display: flex;
   flex-direction: column;
```
Comment (zieeco) [inline style.css:622]: **[REQUIRED CHANGES]**

- Please, kindly try reducing the height to reduce the too much space to match the Figma design below:

*figma*
small-space

*your design*
big-space

File: style.css [hunk 7]
```diff
@@ -601,3 +794,477 @@ button:hover {
...
+  }
+
+  .lascs {
+    position: absolute;
+    left: 297px;
+    top: 84px;
+    width: 122px;
+    height: 120px;
+  }
+
```
Comment (zieeco) [inline style.css]: **[REQUIRED CHANGES]**

- Please, kindly improve your styling on each skill to match the Figma design as much as you can below:

*your design*
skillss-s


*figma*
skills

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @0zedd 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Nice design :heavy_check_mark: 
- Linters are passing :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 8]
```diff
@@ -601,3 +826,514 @@ button:hover {
...
+  .sec_abt {
+    margin-left: 142px;
+  }
+
+  .contact_section2 {
+    margin-left: 147px;
+    margin-top: 0;
+  }
+
+  .abt_icn {
```
Comment (topeogunleye) [inline style.css]: - [ ] As suggested by the previous reviewer, I kindly suggest that you ensure that ALL your social media icons have the same width as the Figma design below:

| Your Design  | Template Design |
| ------------- | ------------- |
| Screenshot from 2022-06-11 12-49-18 | Screenshot from 2022-06-11 12-56-52 |

File: style.css [hunk 9]
```diff
@@ -14,6 +14,10 @@
   justify-content: space-between;
 }
```
Comment (topeogunleye) [inline style.css:18]: - [ ] As suggested by the previous reviewer,  I kindly suggest that you make use of the font used in your template design which is [poppins](https://fonts.google.com/specimen/Poppins). 

You can import Poppins into your CSS file by writing this at the top of your CSS file:
`@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&display=swap');
`

File: style.css [hunk 10]
```diff
@@ -253,9 +321,6 @@ ul {
...
 
 button {
```
Comment (topeogunleye) [inline style.css:325]: - [ ] As suggested by the Previous Code Reviewer, I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

Screenshot from 2022-04-27 19-12-50

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @0zedd,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ishimwezachee — APPROVED** [review]
Hi @0zedd , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**